src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -132,7 +132,6 @@ Copyright (c) .NET Foundation. All rights reserved.
                                        RuntimeConfigDevPath="$(ProjectRuntimeConfigDevFilePath)"
                                        RuntimeIdentifier="$(RuntimeIdentifier)"
                                        PlatformLibraryName="$(MicrosoftNETPlatformLibrary)"
-                                       RuntimeFrameworkVersion="$(RuntimeFrameworkVersion)"
```

Comment:
I'm no fan of dead code.  I feel putting in things "just in case we need it" leads to more confusion over time than its worth.  For example, if someone new comes in to this class to fix a bug, and starts looking where this option is used, doesn't see it, and now starts looking through commit history to find out why it is there.